src/app/globals.css

```
@@ -1 +1,19 @@
 @import 'tailwindcss';
+@keyframes scanSweep {
```

Stylelint (17.7.0)

[error] Expected keyframe name "scanSweep" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)